Source/mpq/mpq_sdl_rwops.cpp

```diff
@@ -72,10 +72,10 @@ static OffsetType MpqFileRwSeek(struct SDL_RWops *context, OffsetType offset, in
 	}
 
 	if (newPosition == data.position)
-		return static_cast<OffsetType>(newPosition);
+		return newPosition;
 
 	if (newPosition > data.size) {

```

Comment:
Looks like MSVC doesn't like this since it's now signed vs unsiged. Other then that the changes looks good :+1: